bot/bot.py

```diff
@@ -3,18 +3,18 @@
 import discord
 from discord.ext import commands
 
-from helpers.api import ApiHelper
+from . import cogs
+from . import helpers
```

Comment:
What's the difference between this and `import cogs`?

Reply:
So I realized I typed the import statement out wrong originally, and the only reason it worked was because the folder was in my pythonpath environment variable. Since adding the bot folder to your pythonpath was a part of the setup nobody had issues. Because the DB required me to change how the bot runs for the better, I removed the pythonpath requirement and broke the original import 😳. So I had to do it correctly here.

Reply:
Ahh okay, definitely a step in the right direction then :)